rust/datafusion/src/physical_plan/parquet.rs

```diff
@@ -665,7 +661,7 @@ fn build_statistics_array(
         statistics
     } else {
         // no row group has statistics defined
-        return build_null_array(data_type, statistics_count);
+        return new_array_with_nulls(data_type, statistics_count);
```

Comment:
Not sure whether you want to change it in this PR, but there is some code that could use `new_array_with_nulls` too in the function `to_array_of_size`, which uses this:
`Arc::new(repeat(None).take(size).collect::<Float64Array>())`

Reply:
I'll change it, thanks :)